src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="month month--start-day--sunday month--month-length--29">
```

Comment:
```suggestion
    <div class="calendar calendar--sunday calendar--29">
```
В таком виде можно записать